backends/client.go

```diff
@@ -1,8 +1,12 @@
 package backends
 
+import (
+	"github.com/bndw/pick/crypto"
+)
+
 type Client interface {
-	Load() ([]byte, error)
-	Save([]byte) error
+	Load() (crypto.Config, []byte, error)
```

Comment:
I'd strongly prefer to not add the `crypto.Config` to the backend client interface. I feel this introduces complexity, by giving the backend client responsibility of marshaling, encoding, and/or dictating the serialization format. 

If we need to encode the `crypto.Config` in the at rest data, which I think we do since we're supporting multiple formats, the serialization format should be owned by the `Safe`. Otherwise, we'll have to reimplement this logic for each backend (e.g. A Cloud Provider), and then maintain that.

By maintaining a strict and minimal backend client interface that simply reads and writes bytes,  adding additional backends will be cheap.